pkg/kafka/testbed/gobgp/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.16.3
```

Comment:
move all gobgp related changes in a separate PR once you complete multi node pipeline with xrd

Reply:
will do, not relevant to this PR then